usersDb.js

```diff
@@ -0,0 +1,69 @@
+const pgp = require('pg-promise')()
```

Comment:
This file should be inside a folder called `db`. So the path of this file would be `/db/user.js`. That way you can get rid of the `Db` in the name of the file. This makes for better code organization. In the future if you have more tables in your database, then the files for the CRUD functions would go inside the `db` folder